traitsui/qt4/ui_panel.py

```diff
@@ -393,8 +393,8 @@ def show_help_popup(event):
     # of the object with the 'help' trait):
     help = getattr(control, "help", None)
     if help is not None:
-        html = template.item_html % (control.GetLabel(), help)
-        HTMLHelpWindow(control, html, 0.25, 0.13)
+        html_content = template.item_html % (control.GetLabel(), help)
+        HTMLHelpWindow(control, html_content, 0.25, 0.13)
```

Comment:
Not here, but in the `HTMLHelpWindow.__init__` method, there is still a parameter named `html`

Reply:
I had made this change previously but Rahul suggested we avoid changing the public api to be safe
https://github.com/enthought/traitsui/pull/1540#discussion_r599310531

Reply:
I'm okay with making this change now. I searched through github and there seem to be no direct users of the traitsui class `HTMLHelpWindow` - https://github.com/search?p=1&q=HTMLHelpWindow+extension%3Apy&type=Code

Reply:
Ah right. Keyword args.